### PR TITLE
Event collection

### PR DESCRIPTION
**Pull request by yara-58**

### CODE REVIEW

File: WPF/SeeShells/SeeShells/UI/Test.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+﻿using System;
```
Comment (AlekStoyanov) [inline WPF/SeeShells/SeeShells/UI/Test.cs]: This class needs to be deleted, It was just to test on the fly, but now there is a proper unit test.

File: WPF/SeeShells/SeeShells/UI/Test.cs [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+{
+    public class Test
+    {
+        public Test()
+        {
+            List<IEvent> evenList = eventCollection.get();
+
+        }
+    }
+}
```
Comment (AlekStoyanov) [inline WPF/SeeShells/SeeShells/UI/Test.cs]: This class needs to be deleted, It was just to test on the fly, but now there is a proper unit test.

File: WPF/SeeShells/SeeShellsTests/SeeShellsTests.csproj [hunk 3]
```diff
@@ -64,6 +65,9 @@
     <Reference Include="System.Transactions" />
     <Reference Include="System.Web" />
     <Reference Include="System.Xml" />
+    <Reference Include="WindowsBase, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
+      <HintPath>..\packages\WindowsBase.4.6.1055.0\lib\WindowsBase.dll</HintPath>
```
Comment (klaki892) [inline WPF/SeeShells/SeeShellsTests/SeeShellsTests.csproj]: Where'd this dependency come from?

Comment (AlekStoyanov) [reply WPF/SeeShells/SeeShellsTests/SeeShellsTests.csproj]: I think I saw it popping up as a suggestion from visual studio, and @yara-58 must have installed it accidentally. We do not need it.

Comment (AlekStoyanov) [reply WPF/SeeShells/SeeShellsTests/SeeShellsTests.csproj]: The dependency should be removed from NuGet

Comment (yara-58) [reply WPF/SeeShells/SeeShellsTests/SeeShellsTests.csproj]: yes i did accidentally install that

File: WPF/SeeShells/SeeShellsTests/packages.config [hunk 4]
```diff
@@ -3,6 +3,7 @@
   <package id="MockHttpServer" version="1.4.0.0" targetFramework="net46" />
   <package id="MSTest.TestAdapter" version="1.3.2" targetFramework="net45" />
   <package id="MSTest.TestFramework" version="1.3.2" targetFramework="net45" />
+  <package id="WindowsBase" version="4.6.1055.0" targetFramework="net46" />
```
Comment (AlekStoyanov) [inline WPF/SeeShells/SeeShellsTests/packages.config]: WindowsBase package is still here.

File: WPF/SeeShells/SeeShellsTests/UI/EventCollectionTest.cs [hunk 5]
```diff
@@ -0,0 +1,26 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using SeeShells;
+using SeeShells.UI;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SeeShellsTests.UI
```
Comment (AlekStoyanov) [inline WPF/SeeShells/SeeShellsTests/UI/EventCollectionTest.cs]: It would be nice if you add docs to this.

Comment (yara-58) [reply WPF/SeeShells/SeeShellsTests/UI/EventCollectionTest.cs]: I added it in the previous one i deleted and forgot to put it in this file, I will add it now

File: WPF/SeeShells/SeeShellsTests/packages.config (hunk 4 above)
Comment (AlekStoyanov) [inline WPF/SeeShells/SeeShellsTests/packages.config]: WindowsBase package is still here.

**Pull request merged**